lib/ui/profile/interactor/profile_data.dart

```diff
@@ -17,7 +17,7 @@ class ProfileData {
 
   factory ProfileData.fromJson(Map<String, dynamic> json) {
     final publicData = json['publicData'];
-    final name = publicData['name'];
+    final name = publicData['name'] ?? '';
```

Comment:
can this name be null? 
If so, we should make the name nullable instead of faking a empty string

its ok to use null.